packages/core/src/input/InputManager.ts

```diff
@@ -0,0 +1,375 @@
+import { Ray, Vector2 } from "@oasis-engine/math";
+import { Engine } from "../Engine";
+import { Entity } from "../Entity";
+import { HitResult } from "../HitResult";
+import { PhysicsManager } from "../PhysicsManager";
+import { SceneManager } from "../SceneManager";
+import { Script } from "../Script";
+import { Input } from "./Input";
+import { Pointer } from "./Pointer";
+
+/** @internal */
+enum PointerEventType {
+  PointDown = 0,
+  PointUp = 1,
+  PointMove = 2
+}
+
+/** @internal */
+enum PointerChangeType {
+  Update = 0,
```

Comment:
No need for `Update = 0`, `Update` is ok.

Reply:
Done.